README.md

```diff
@@ -1,3 +1,15 @@
 # Path to Craftshmanship
 
 Ideas on getting to be a software craftsman. Practical lessons.
+
+# General Principles
+
+The following are principles aimed at harmonizing the documentation which is
+yet to be written.
+
+- always use the right terminology
```

Comment:
Agreed. We should not create confusion with different terms than the ones already largely used. For example we should not deliberately avoid using certain terms just for the sake of lowering the discussion level (such an opinion expressed in Romanian: https://devforum.ro/t/what-i-learned-from-working-in-failed-platforms/2320/26)